[2024-04-13T17:18:45.416+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-13T17:18:45.454+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-13T17:17:33.443854+00:00 [queued]>
[2024-04-13T17:18:45.471+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.get_files manual__2024-04-13T17:17:33.443854+00:00 [queued]>
[2024-04-13T17:18:45.472+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-04-13T17:18:45.499+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_files> on 2024-04-13 17:17:33.443854+00:00
[2024-04-13T17:18:45.517+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'get_files', 'manual__2024-04-13T17:17:33.443854+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpl92vm12b']
[2024-04-13T17:18:45.520+0000] {standard_task_runner.py:91} INFO - Job 11: Subtask get_files
[2024-04-13T17:18:45.547+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=42903) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-13T17:18:45.554+0000] {standard_task_runner.py:63} INFO - Started process 42924 to run task
[2024-04-13T17:18:45.585+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.get_files manual__2024-04-13T17:17:33.443854+00:00 [running]> on host 2ce858e27684
[2024-04-13T17:18:45.714+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='get_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-13T17:17:33.443854+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-13T17:17:33.443854+00:00'
[2024-04-13T17:18:45.716+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-13T17:18:45.717+0000] {logging_mixin.py:188} INFO - 1 files found in /opt/***/dags/data
[2024-04-13T17:18:45.717+0000] {python.py:237} INFO - Done. Returned value was: ['/opt/***/dags/data/order_bazilian.csv']
[2024-04-13T17:18:45.718+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-13T17:18:45.754+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=get_files, execution_date=20240413T171733, start_date=20240413T171845, end_date=20240413T171845
[2024-04-13T17:18:45.785+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-13T17:18:45.815+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-13T17:18:45.828+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
